inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -740,69 +767,91 @@ linelist_cleaned <- linelist_cleaned %>%
+factor_vars
+
+This chunk will help you clean factor variables.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+```{r factor_vars}
+
 # Force missing values to NA
 # important for sex to generate age pyramids
 linelist_cleaned$sex <- fct_recode(linelist_cleaned$sex, 
                                    NULL = "Unknown/unspecified")
 
 
 # change the order of levels in a single categorical variable 
+# This orders the levels -- since in all figures / tables, 0-4 should come
+# before 4-24, etc
 linelist_cleaned <- linelist_cleaned %>% 
   mutate(time_to_death = factor(time_to_death, 
                                 levels = c("0-4 hours", 
                                            ">4-24 hours", 
                                            ">24-48 hours", 
                                            ">48 hours")))
 
-# Change the order of levels of multiple categorical variables
+
+
+# Change the order of levels of multiple categorical variables at the same time
 linelist_cleaned <- linelist_cleaned %>%
   mutate_at(vars(starts_with("test")),          # Looks for variables beginning with "test"
             fct_relevel,
             "Positive", "Negative", "Not done"  # Sets order of levels
            )
+```
+
 
-# Create a variable based on rules from other simple character variables
-# If you have access to lab results, you can create a case definition variable 
-# the tilda (~) is used to assign the new values (Conf, prob, susp, unknown)
-# starting from the specific to the general
-# TRUE assigns all remaining rows 
-# You MUST modify this section to match your case definition. The below
-# uses positive RDT for Confirmed and epi link only for Probable.
-#
-linelist_cleaned <- linelist_cleaned %>%
-  mutate(case_def = case_when(
-    is.na(hep_e_rdt) & is.na(other_cases_in_hh)           ~ NA_character_,
-    hep_e_rdt == "Positive"                               ~ "Confirmed",
-    hep_e_rdt != "Positive" & other_cases_in_hh == "Yes"  ~ "Probable",
-    TRUE                                                  ~ "Suspected"
-  ))
 
-# vectors of variable names ----------------------------------------------------
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+vector_vars
 
-## You may want to group the names of several variables that have the same possible 
-## values in to a named vector. 
-## This way if you want to run the same function over these variables you can 
-## simply use the named vector rather than typing out each variable individually
+This chunk creates groups of variables that you might want to use together. That
+way, if you want to run the same function over these variables, you can run it
+all at once. For example, you may want to look at frequency of all symptoms at
+the same time.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+```{r vector_vars}
 
```

Comment:
I'm thinking that these variables are things that can go in the setup chunk at the top since we are simply defining them.

Reply:
mmmm feel like when you are going through your first analysis and setting up you would be browsing etc first, before you know which vars you want to pull together....

Reply:
agree with alex -- you possibly don't have the variables correctly named or coded, etc.